docs/user/components/maxbin2.rst

```diff
@@ -45,5 +45,11 @@ Default directives
 
 - ``container``: flowcraft/maxbin2
 - ``version``: 2.2.4-1
-- ``cpus``: 3
-- ``memory``: 5.GB * task.attempt
+- ``cpus``: 4
+- ``memory``: 8.GB (dynamically increased on retry)
```

Comment:
Are we now adopting this on the docs? Rather than the task.attempt nomenclature?

Reply:
I check some examples and I saw both... I went to the more descriptive one and made a mental note (now written here) to change them to that.